### DISCUSSION TITLE
Software audio equaliser / EQ

### DISCUSSION THREAD

**Discussion by ProperTeaMogul** [mcguirepr89/BirdNET-Pi#722] [Ideas]
Long-time lurker and finally got this up and running on an RPi 3B+ - it works really well so kudos to the contributors!

My current microphone set up picks up lots of human/road noise below 1kHz. It would be great to be able to reduce these whilst boosting higher frequencies. There is a hardware solution by picking a microphone with a certain frequency response or running through a high-pass filter. However, does anyone know of a software solution on the RPi to equalize the input prior to it running in BirdNET?

I've read about ALSA Equal but can't seem to get that to work and wondered if a feature existed or anyone has had any luck in this area.

**Comment by ehpersonal38** [comment]
Hi

It looks like the following application would fit your needs for system wide real-time audio equalization: https://github.com/wwmm/easyeffects It does seem to be GUI only though, so you'll probably have to boot your Pi up with a keyboard and monitor to configure it and such

 (edit: just remembered most of us are running Raspberry Pi OS Lite, not sure how that would work then honestly. Maybe you could configure easyeffects on a Raspberry Pi with a GUI then export it to a preset file, then load it on your BirdNET-Pi [via command line](https://github.com/wwmm/easyeffects/wiki/Command-Line-Options)?. Or does VNC/Xserver work with OS Lite? Ah I can't think very well today, oh well, hopefully I've given you some leads that may be useful).

**Reply by ProperTeaMogul** [reply]
That is a good pointer! A bit of googling suggests the following steps:
1. Set up RPi to a screen and keyboard
2. Download EasyEffects
3. Set up GUI (sites seem to suggest X or Xvfb)
4. Boot GUI
5. Play with EasyEffect settings
6. Reboot BirdNET-Pi and see how it affects recordings
7. Repeat 4-6 as necessary.

I'm a bit of a novice with this and installed EasyEffects, unfortunately that seems to have stopped recording completely in BirdNET-Pi! Argh!
